packages/builder/src/utils/wagmi.ts

```diff
@@ -133,24 +129,25 @@ function enabledChainsAndProviders() {
   if (!usingDevOnlyChains) {
     if (
       process.env.NODE_ENV !== "test" &&
-      (infuraId === "" || alchemyId === "")
+      (config.blockchain.infuraId === undefined ||
+        config.blockchain.alchemyId === undefined)
     ) {
       throw new Error(
         "REACT_APP_INFURA_ID and REACT_APP_ALCHEMY_ID must be set to use non-local chains"
```

Comment:
Could this be done in `getConfig`? Would probably also help get rid of the non-null assertions a couple of lines below.